examples/confirm-command.ts

```diff
@@ -0,0 +1,22 @@
+import {Command, ux} from '@oclif/core'
+
+import {confirm} from '../src/ux/confirm.js'
+
+export default class ConfirmCommand extends Command {
+  static description = 'Example command demonstrating confirm usage'
+
+  async run() {
+    const shouldProceed = await confirm('Do you want to proceed with this action?')
+
+    if (shouldProceed) {
+      ux.stdout('User confirmed the action!')
+    } else {
+      ux.stdout('User declined or timed out.')
+    }
```

Comment:
Not a blocker, but we should add something here to end the process after either of these options are selected. Right now I have to `ctrl+c` out, they just hang.

Reply:
oh yeah i noticed that too and forgot. thanks, i'll update it.